.github/workflows/ci.yaml

```
@@ -0,0 +1,134 @@
+          name: built-bdist
+
+  release:
+    needs:
+      - build
+      - sonarcloud
+    # upload to PyPI on every tag and ensure it does not run on forks
+    if: github.event_name == 'push' && github.ref_type == 'tag' && !github.event.pull_request.head.repo.fork
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: ""
```

actionlint (1.7.7)

string should not be empty

(syntax-check)